opt/api/README.md

```diff
@@ -0,0 +1,23 @@
+## WebServer
+
+## Run Locally
+
+### With Docker
+
+1. Build the ./DockerFile: `docker build ../.. -t enguage -f Dockerfile`
```

Comment:
Run this within `opt/api` then you should be able to run `curl -X POST -H "Content-Type: application/json" -d '{"sessionId": "1", "utterance": "what do I need?"}' http://localhost:8080/interpret`

Reply:
Yep, that's brill!